src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackageExtractionContext.cs

```diff
@@ -15,24 +15,25 @@ public class PackageExtractionContext
 
         public XmlDocFileSaveMode XmlDocFileSaveMode { get; set; }
 
-        public IPackageSignatureVerifier SignedPackageVerifier { get; set; }
-
-        public SignedPackageVerifierSettings SignedPackageVerifierSettings { get; set; }
+        public ClientPolicyContext ClientPolicyContext { get; }
 
         public bool CopySatelliteFiles { get; set; } = true;
 
+        /// <remarks>
```

Comment:
nit: Why move this below CopySatelliteFiles?

Reply:
I wanted to have `public` attributes together and since `SignedPackageVerifier` is now `internal` I moved it... I can return it to the previous place.

Reply:
Nah, it's fine. 
It makes sense. Just wanted to udnerstand your train of thought and I missed the access modifier change.